### PR TITLE
fix: Some links are broken at daytona.io documentation page

### LINKED ISSUE #92: Some links are broken at daytona.io documentation page
**Describe the bug**
Some links are broken at daytona.io documentation page 

**To Reproduce**
1. Go to 'https://www.daytona.io/docs/installation/installation/'
2. 1st broken link: [add a git provider](https://www.daytona.io/docs/installation/configuration/git-providers#add-a-git-provider) on front page.
3. 2nd broken link: [use the cli](https://www.daytona.io/docs/installation/reference/cli) on front page.
4. 3rd broken link [builders](https://www.daytona.io/docs/usage/usage/builders)
5. Click on it '
6. See error page.

**Expected behavior**
1) correct first link: [Go to cli ](https://www.daytona.io/docs/reference/cli/) refer here.
2) correct second link: [add a git provider](https://www.daytona.io/docs/configuration/git-providers/) refer here.
3) correct second link: [builders](https://www.daytona.io/docs/usage/builders/) refer here.

**Screenshots**
image

**Desktop (please complete the following information):**
 - Any
 - Daytona Version: Documentation [v0.21]

**Additional context**
Will add more broken links as soon as possible.

### PR DESCRIPTION
**Pull request by Anshgrover23**
# Some links are broken at daytona.io documentation page

# Description:

- This pull request addresses issues with relative links in the documentation. Specifically, it removes the usage of `..` in all paths, replacing them with absolute paths to ensure that links work correctly in both development and production environments.

# Related Issues:
Fixes #92 

# Screenshots:

Before : Screenshot (112)

After: 
Screenshot (111)

### CODE REVIEW

**Review by Tpuljak — CHANGES_REQUESTED** [review]
https://github.com/user-attachments/assets/e9386f84-4e47-4ab8-987e-abe6b77a72ad

@Anshgrover23 did you test the changes before opening the PR?

**Comment by Anshgrover23** [comment]
@Tpuljak bro i think there is some misunderstanding. i have changed the links of installation page not the provider or any specific page. and yes i have tested that so to added the screenshot. 
# Before video:
https://github.com/user-attachments/assets/fe66ba41-c213-4a09-99b0-09f24aa791c9
# after video:
https://github.com/user-attachments/assets/dc5b983f-70ca-4a4e-8044-96ac4910945f

**Comment by Tpuljak** [comment]
> @Tpuljak bro i think there is some misunderstanding. i have changed the links of installation page not the provider or any specific page. and yes i have tested that so to added the screenshot.

https://github.com/daytonaio/docs/pull/94/files#diff-1cbe5d078b290d92561c02bf6f7a2d2e5b2877d86b518c6b4c0dd8ea4e3a655a

Then why does your diff suggest that you changed other links?

**Comment by Anshgrover23** [comment]
changes that file also see video:

https://github.com/user-attachments/assets/746cb443-acaf-46a9-9b2e-9050faeb9ea7

File: src/content/docs/configuration/providers.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ Daytona abstracts the management and deployment of Workspaces into 2 related con
     Daytona allows you to configure new and existing Targets, and choose between them at Workspace creation time.
 
 Daytona's architecture decentralizes the role of a Provider into a separate service.
-The [Daytona Server](../reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
```
Comment (Tpuljak) [inline src/content/docs/configuration/providers.mdx:23]: Please test **ALL** the links you changed.

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: src/content/docs/configuration/providers.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Aside from '@components/Aside.astro'
 Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
 
 1. **Provider**  
-    A plugin that interfaces with the [Daytona Server](../reference/server), responsible for Workspace deployment and lifecycle management.
+    A plugin that interfaces with the [Daytona Server](/usage/server), responsible for Workspace deployment and lifecycle management.
```
Comment (Anshgrover23) [inline src/content/docs/configuration/providers.mdx:13]: @Tpuljak I have already changed and tested. It's already working. See video above.

File: src/content/docs/configuration/providers.mdx (hunk 1 above)
Comment (Anshgrover23) [reply src/content/docs/configuration/providers.mdx:23]: Resolved already. See properly.

Comment (Tpuljak) [reply src/content/docs/configuration/providers.mdx:23]: https://github.com/user-attachments/assets/09556f70-1e6d-4561-93b1-32a12e930a9e

You're really testing me here 😄 

I'll close the PR if you request a new review without testing **ALL** the links you changed.

Comment (Anshgrover23) [reply src/content/docs/configuration/providers.mdx:23]: Ok 👌 I will check **All** the links 😃 and then make a review

**Comment by Anshgrover23** [comment]
@Tpuljak ready for review and merge sir.

**Review by Tpuljak — APPROVED** [review]
Nice work. Thanks!

**Comment by Anshgrover23** [comment]
@Tpuljak when will it get merged?

**Comment by Tpuljak** [comment]
> @Tpuljak when will it get merged?

When another maintainer gives their review

**Review by ivan-burazin — APPROVED** [review]

**Comment by Tpuljak** [comment]
@Anshgrover23 please rebase and resolve the conflict before we merge

File: src/content/docs/usage/workspaces.mdx [hunk 3]
```diff
@@ -37,13 +37,19 @@ Creating a Workspace allows you to set up a new environment either by selecting
    daytona create
    ```
 
+
+2. Select the Git Provider and repository you want your Workspace to be configured with.
```
Comment (Tpuljak) [inline src/content/docs/usage/workspaces.mdx:41]: We removed these sections from the workspaces page in https://github.com/daytonaio/docs/pull/96

Pleas only include the fixes to the links in this PR

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Review by Tpuljak — APPROVED** [review]

**Pull request merged**